cli/printer/text.go

```diff
@@ -85,11 +89,12 @@ func (t *TextPrinter) PrintEnvironments(environments ...*models.Environment) err
 		return e.Links[i]
 	}
 
-	rows := []string{"ENVIRONMENT ID | ENVIRONMENT NAME | OS | SCALE | INSTANCE TYPE | LINKS"}
+	rows := []string{"ENVIRONMENT ID | ENVIRONMENT NAME | TYPE | OS | SCALE | INSTANCE TYPE | LINKS"}
```

Comment:
[nit] I would argue that `INSTANCE_TYPE` should be dropped from this table - the table is getting pretty large and I think it is the least viewed/important field here.

Reply:
Agree that this is getting large. If we remove this, I think we should drop SCALE as well.

SCALE, INSTANCE_TYPE will be a lot less relevant going forward but would you we want the user to resort to showing the json output to check these fields?

Reply:
I like the idea of dropping SCALE as well since it makes static environments feel a little more "legacy" and discouraged. With that in mind, I think it's ok to require users's specify json output to see that.

Reply:
✅